### ISSUE TITLE
Seeking clarification on pronunciation of this library's name

### ISSUE THREAD

**Issue by thebanjomatic** [sindresorhus/ky#635]
My team has been discussing using this library, but there has been some debate on what the intended pronunciation is. Is it Ky (pronounced `/kiː/` like “Key”), or Ky (pronounced `/kaɪ/` like “Kyle”) or just `/keɪ waɪ/` as in “K.Y.”

_Literally_ every time the library comes up its "should we use got or kai or key or k.y. or whatever it's called."

Can you please settle this debate (and maybe consider adding the pronunciation to the readme)

**Comment by sholladay** [comment]
It is an abbreviation of "kūki yomenai". So technically you should pronounce it by saying the names of the letters individually, as in K.Y.

However, I tend to pronounce it like "sky", but without the "s", just because it is easier and quicker to say.

There, you have reasons to say it either way! 😆

For what it's worth, I also pronounce Deno as "dehno", like Ry did originally, and not "deeno", which is official now. I'm a rebel.

**Comment by thebanjomatic** [comment]
Yup, abbreviations often times spelled out, but at least in the tech world they often are pronounced as well: SQL, GIF, JPEG, GNU, etc.

**Comment by sholladay** [comment]
True and that's part of why I don't bother to use the spelling pronunciation. But KY is not a tech abbreviation, we didn't create the abbreviation. It's an existing Japanese slang term that means "unable to read the air", which happens to be an amusing name for a fetch library.
